utils/.env.template

```diff
@@ -0,0 +1,36 @@
+# Environment variables
+
+### IMPORTANT: when done editing this file, rename it to ".env" 

```

Comment:
You might want to take a look in [Createnv](https://github.com/cuducos/createnv) to easily automate that.

 **Disclaimer**: I am the creator of that package! But, I mean… I wrote it exactly to make it easy to onboard newcomers to projects using `.env` files ✨